### PR TITLE
Use the DBus Storage module

### PR DESCRIPTION
**Pull request by poncovka**
The local storage object is no longer available. Use the DBus module instead.

**Depends on:** https://github.com/rhinstaller/anaconda/pull/2331

### CODE REVIEW

**Comment by redhatrises** [comment]
@openscap-ci test this please

**Comment by redhatrises** [comment]
@poncovka Thanks for the patch! Can you fix the import error?

**Comment by poncovka** [comment]
@redhatrises, the import error says `ImportError: cannot import name 'nm' from 'pyanaconda'`. The pull request https://github.com/OpenSCAP/oscap-anaconda-addon/pull/106 should fix this issue. If you merge it, I can fix the remaining issues here.

**Comment by redhatrises** [comment]
@poncovka #106 has been merged. If you can fix the remaining issues, that would be great! Thanks!

**Comment by dahaic** [comment]
@openscap-ci test this please

**Comment by poncovka** [comment]
Hi, I have looked at the failing tests and a lot of stuff was already fixed in the rhel8-branch, but never backported to upstream. Could you backport these commits, please?

**Comment by redhatrises** [comment]
@poncovka can you rebase?

**Comment by poncovka** [comment]
The tests are updated.

**Comment by redhatrises** [comment]
@openscap-ci test this please

**Comment by poncovka** [comment]
Hi, the tests from `tests/test_rule_handling.py` are failing, because they require a newer version of Anaconda. They pass when I run them locally with the latest version.

The tests from `tests/test_content_handling.py` are failing with a message `Invalid SCAP Source Datastream`. It doesn't seem to be related to this PR.

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
ok to test

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
We are experiencing CI issues with this project, so "test this please" and other CI-related comments are being added, and subsequently deleted.

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
ok to test

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
test this please

File: tests/test_rule_handling.py [hunk 1]
```diff
@@ -113,7 +116,46 @@ def storage_mock():
     return mock.Mock()
 
 
-@mock.patch("pyanaconda.dbus.DBus.get_proxy")
+@pytest.fixture()
+def proxy_getter(monkeypatch):
```
Comment (matejak) [inline tests/test_rule_handling.py:120]: Could you please clarify what is going on here? I think that you have replaced the need for a mock injection on a per-test case basis to injecting a mock factory at the top, but I am not sure how exactly it works.
Fixtures with arguments are a less-known feature of `pytest`, so we have to figure out how to describe this, so developers are not confused and they can continue writing tests in the same spirit.

**Comment by matejak** [comment]
Merging, LGTM, other outstanding issues can be fixed in #111 and following PRs.

**Pull request merged**